playwright/src/main/java/com/microsoft/playwright/impl/APIRequestImpl.java

```diff
@@ -65,7 +66,8 @@ public APIRequestContextImpl newContext(NewContextOptions options) {
       params.add("storageState", storageState);
     }
     addToProtocol(params, clientCertificateList);
-    JsonObject result = playwright.sendMessage("newRequest", params).getAsJsonObject();
+    // Timeout here is not the command's timeout, but we pass it for consistency.
+    JsonObject result = playwright.sendMessage("newRequest", params, options.timeout).getAsJsonObject();
```

Comment:
Let's pass `NO_TIMEOUT` instead? Otherwise we might get unexpected timeouts later on, when server starts supporting the timeout here.

Reply:
Ok, fixed as discussed.